libs/getdns/Makefile

```diff
@@ -59,16 +59,6 @@ CMAKE_OPTIONS += -DENABLE_STUB_ONLY=$(if $(CONFIG_GETDNS_ENABLE_STUB_ONLY),ON,OF
 CMAKE_OPTIONS += -DUSE_LIBIDN2=$(if $(CONFIG_GETDNS_ENABLE_IDN_LIBIDN2),ON,OFF)
 
 
-define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/getdns/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/getdns/getdns*.h $(1)/usr/include/getdns/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libgetdns*.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/getdns*.pc $(1)/usr/lib/pkgconfig/
```

Comment:
Check the generated pkgconfig files to make sure libdir points to the prefix and not the host (that is, ${prefix}/lib instead of /usr/lib).

Reply:
Here's what I see in `staging_dir/target-x86_64_musl/usr/lib/pkgconfig/getdns.pc`

```
prefix=/usr
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: getdns 
Version: 1.6.0
Description: A modern asynchronous DNS library

Libs: -L${libdir} -lgetdns
Cflags: -I${includedir}
```

Reply:
Looks correct.